app/helpers/common/priceFormatter.test.js

```diff
@@ -1,26 +1,50 @@
-import { formatPrice, formatDecimal } from './priceFormatter';
+import { formatPrice, formatNumber, removeCommas } from './priceFormatter';
 
 describe('formatPrice', () => {
   it.each`
-    priceValue   | expected
-    ${10}        | ${'10.00'}
-    ${1000}      | ${'1,000.00'}
-    ${999.999}   | ${'999.99'}
-    ${1234.560}  | ${'1,234.56'}
-    ${12.011}    | ${'12.01'}
-  `('formatPrice $priceValue should be formatted to $expected', ({ priceValue, expected }) => {
-    expect(formatPrice(priceValue)).toEqual(expected);
+    value          | minimumFractionDigits    | maximumFractionDigits | expected
+    ${10}          | ${0}                     | ${2}                  | ${'10'}
+    ${10.00}       | ${0}                     | ${4}                  | ${'10'}
+    ${10.10}       | ${0}                     | ${2}                  | ${'10.10'}
+    ${10}          | ${2}                     | ${2}                  | ${'10.00'}
+    ${10.0001}     | ${2}                     | ${4}                  | ${'10.0001'}
+    ${999.999}     | ${2}                     | ${4}                  | ${'999.999'}
+    ${1234.560}    | ${2}                     | ${4}                  | ${'1,234.56'}
+    ${1234.56019}  | ${2}                     | ${4}                  | ${'1,234.5602'}
+    ${12.011}      | ${2}                     | ${2}                  | ${'12.01'}
+    ${12}          | ${5}                     | ${5}                  | ${'12.00000'}
+    ${12.0}        | ${1}                     | ${5}                  | ${'12.0'}
+  `('formatPrice $value should be formatted to $expected', ({
+    value, minimumFractionDigits, maximumFractionDigits, expected,
+  }) => {
+    expect(formatPrice({ value, minimumFractionDigits, maximumFractionDigits })).toEqual(expected);
   });
 });
 
-describe('formatDecimal', () => {
+describe('formatNumber', () => {
   it.each`
-  priceValue   | expected
-  ${10}        | ${10}
-  ${10.1}      | ${'10.10'}
-  ${12.34}     | ${'12.34'}
-  ${999.999}   | ${999.999}
-`('formatDecimal $priceValue should be formatted to $expected', ({ priceValue, expected }) => {
-    expect(formatDecimal(priceValue)).toEqual(expected);
+  priceValue        | expected
+  ${10}             | ${'10'}
+  ${10.1}           | ${'10.1'}
+  ${12.34}          | ${'12.34'}
+  ${999.999}        | ${'999.999'}
+  ${999.9999}       | ${'999.9999'}
+  ${999.99999999}   | ${'1,000'}
+  ${999.1111111}    | ${'999.1111'}
+  ${999.123456789}  | ${'999.1235'}
+`('formatNumber $priceValue should be formatted to $expected', ({ priceValue, expected }) => {
+    expect(formatNumber({ value: priceValue })).toEqual(expected);
+  });
+});
```

Comment:
Sorry, I should have notice this one yesterday.

Given that we're testing `formatNumber` here `value` is probably a better name than `priceValue`. Let's make that the last change and then we can close the PR.